lacework/readme.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Vulnerability scanning with Lacework
```

Comment:
For consistencies sake, perhaps this should be more like the Snyk template? Or perhaps that should be updated to be more inline with this one. 

e.g.
```suggestion
title: Scan project using Lacework
```

<img width="336" alt="Screenshot 2024-05-15 at 1 12 54 pm" src="https://github.com/buildkite/templates/assets/656826/ec309e2d-5965-4fe5-b9f5-aeef256b5525">